spec/season_statistics_spec.rb

```diff
@@ -23,19 +23,32 @@
     end
   end
 
-#   describe "#winningest_coach" do
-#     xit " names the Coach with the best win percentage for the season" do 
-
-#       # return value string
-#     end
-#   end
-
-#   describe "#worst_coach" do
-#     xit " names the Coach with the worst win percentage for the season" do 
+  describe "#coach_record" do 
+  it "can list the loss, ties, and wins of each coach" do 
+    expected = {"Adam Oates" => {:games_coached=>183, :losses=>79, :ties=>44, :wins=>60},
+                "Alain Vigneault" => {:games_coached=>523, :losses=>192, :ties=>100, :wins=>231},
+                "Barry Trotz" => {:games_coached=>521, :losses=>179, :ties=>99, :wins=>243},
+                "Bill Peters" => {:games_coached=>328, :losses=>141, :ties=>62, :wins=>125},
+                "Bob Boughner" => {:games_coached=>82, :losses=>28, :ties=>16, :wins=>38},
+                "Bob Hartley" => {:games_coached=>305, :losses=>148, :ties=>52, :wins=>105},
+                "Bruce Boudreau" => {:games_coached=>511, :losses=>161, :ties=>108, :wins=>242}}
+    # expect(@season_stats.coach_record).to eq(expected)
+  end
+end
+  
+  describe "#winningest_coach" do
+    it " names the Coach with the best win percentage for the season" do 
+      expect(@season_stats.winningest_coach).to be_a(String)
+      expect(@season_stats.winningest_coach).to eq("Dan Lacroix")
+    end
```

Comment:
in the spec_harness it looks like we're supposed to pass in an argument of season id and calculate the winningest coach for each season. 

here's the assertions from the spec harness for this test:
it "#winningest_coach" do
    expect(@stat_tracker.winningest_coach("20132014")).to eq "Claude Julien"
    expect(@stat_tracker.winningest_coach("20142015")).to eq "Alain Vigneault"
  end

Reply:
Fixed!